### ISSUE TITLE
Cannot create a VRF with the required values using provider version 0.10.0

### ISSUE THREAD

**Issue by jgomezve** [CiscoDevNet/terraform-provider-mso#213]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v1.4.5

### MSO version

* NDO 4.1.1i

### APIC version and APIC Platform for Site Level Resources

* 5.2(7f)

### Affected Resource(s)

* mso_schema_template_vrf

### Terraform Configuration Files

```hcl
terraform {
  required_providers {
    mso = {
      source = "CiscoDevNet/mso"
      version = "0.10.0"
    }
  }
}

provider "mso" {
  insecure = true
  platform = "nd"
}

data "mso_schema" "schema1" {
  name  = "SCHEMA1"
}

resource "mso_schema_template_vrf" "schema_vrf" {
  schema_id    = data.mso_schema.schema1.id
  template     = "TEMPLATE1"
  name         = "VRF1"
  display_name = "VRF1"
}
```

### Debug Output

```console
$ terraform apply -auto-approve
data.mso_schema.schema1: Reading...
data.mso_schema.schema1: Read complete after 0s [id=647785dc8554cfd4b993f17a]

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # mso_schema_template_vrf.schema_vrf will be created
  + resource "mso_schema_template_vrf" "schema_vrf" {
      + display_name           = "VRF1"
      + id                     = (known after apply)
      + ip_data_plane_learning = (known after apply)
      + layer3_multicast       = (known after apply)
      + name                   = "VRF1"
      + preferred_group        = (known after apply)
      + schema_id              = "647785dc8554cfd4b993f17a"
      + template               = "TEMPLATE1"
      + vzany                  = (known after apply)
    }

Plan: 1 to add, 0 to change, 0 to destroy.
mso_schema_template_vrf.schema_vrf: Creating...
╷
│ Error: "invalid Enablement \"\""{}
│ 
│   with mso_schema_template_vrf.schema_vrf,
│   on main.tf line 22, in resource "mso_schema_template_vrf" "schema_vrf":
│   22: resource "mso_schema_template_vrf" "schema_vrf" {
│ 
╵
```
### Expected Behavior

VRF should be configured on the Template

### Actual Behavior
`terraform apply -auto-approve` fails

### Steps to Reproduce

1. `terraform apply -auto-approve`

### Important Factoids

The execution succeeds if the attribute `ip_data_plane_learning` is set:

```hcl
resource "mso_schema_template_vrf" "schema_vrf" {
  schema_id              = data.mso_schema.schema1.id
  template               = "TEMPLATE1"
  name                   = "VRF1"
  display_name           = "VRF1"
  ip_data_plane_learning = "enabled"
}
```

**Comment by akinross** [comment]
Hi Jorge, 

This is a known issue that will be fixed in the next release. The mso-go-client was not updated properly during the release and thus dependency to older version was present in code. See further explanation in: https://github.com/CiscoDevNet/terraform-provider-mso/issues/209. Will keep this issue open until new release is out.

Kind regard,

Akiin

**Comment by marinalf** [comment]
Same here... 

mso_tenant.tenant: Creating...
mso_tenant.tenant: Creation complete after 6s [id=647a8d2a4e49a25d891e1d15]
mso_schema.schema1: Creating...
mso_schema.schema1: Creation complete after 0s [id=647a8d2a4e49a25d891e1d16]
mso_schema_site.azure_site: Creating...
mso_schema_site.aws_site: Creating...
mso_schema_template_vrf.vrf1: Creating...
mso_schema_site.azure_site: Creation complete after 0s [id=64657d602248d1796e3d79b1]
mso_schema_site.aws_site: Creation complete after 1s [id=64657d5b2248d1796e3d79b0]
╷
│ Error: "invalid Enablement \"\""{}
│ 
│   with mso_schema_template_vrf.vrf1,
│   on main.tf line 52, in resource "mso_schema_template_vrf" "vrf1":
│   52: resource "mso_schema_template_vrf" "vrf1" {
│

**Comment by edudppaz** [comment]
@marinalf as a workaround while the new version is commited, you can just put the default value for the ip_data_plane_learning on your resource:

```
resource "mso_schema_template_vrf" "schema_vrf" {
  schema_id    = data.mso_schema.schema1.id
  template     = "TEMPLATE1"
  name         = "VRF1"
  display_name = "VRF1"
  ip_data_plane_learning = "enabled" # temporary fix for bug in v.10
}

```

**Comment by danischm** [comment]
I think this has been fixed in 0.11.0 and can be closed. @akinross and @lhercot to confirm.
